.gitignore

```diff
@@ -46,4 +46,4 @@ crashlytics.properties
 crashlytics-build.properties
 fabric.properties
 .idea
-audit/*.json
```

Comment:
do not remote audit :P 
It contains tons of json files, we do not want to commit